docs/articles/configuration/hocon.md

```diff
@@ -0,0 +1,388 @@
+---
+uid: hocon
+title: Hocon module
+---
+
+# HOCON (Human-Optimized Config Object Notation)
+
+This is an abridged version of HOCON for its use in Akka.NET. A full .NET implementation of HOCON spec can be read [here](https://github.com/akkadotnet/HOCON/blob/dev/README.md)
+
```

Comment:
I would add a very simple example right here - the 20% of basic usage stuff that 80% of users are going to need to know. The rest of the meat of the HOCON spec can be things consumed by users who have more sophisticated use cases or are just more interested in how the technology works.

But otherwise this is looking good thus far.